### DISCUSSION TITLE
Recent upgrade fails

### DISCUSSION THREAD

**Discussion by mbagheri1** [linien-org/linien#402] [General]
I upgraded my operating linen. After the upgrade, the server fails to work with client as the two versions are incompatible (client 2.0.2 and server 1.0.2). Pip install cannot find a server version 2.0.2.

**Comment by bleykauf** [comment]
Did you make sure that you are on RedPitaya OS 2.0?

**Comment by bleykauf** [comment]
Closing this, since this is better discussed in #405.
